app/models/hubstats/repo.rb

```diff
@@ -1,32 +1,31 @@
 module Hubstats
   class Repo < ActiveRecord::Base
 
-    scope :with_recent_activity, lambda {|time| where("updated_at > ?", time).order("updated_at DESC")}
-    scope :with_id, lambda {|user_id| where(id: user_id.split(',')) if user_id}
+    scope :with_recent_activity, lambda {|start_date, end_date| where("hubstats_repos.updated_at BETWEEN ? AND ?", start_date, end_date).order("updated_at DESC")}
+    scope :with_id, lambda {|repo_id| where(id: repo_id.split(',')) if repo_id}
 
-    scope :deploys_or_comments_count, lambda {|time, data, began_time, name|
+    scope :deploys_count, lambda {|start_date, end_date|
       select("hubstats_repos.id as repo_id")
-       .select("IFNULL(COUNT(DISTINCT #{data}.id),0) AS #{name}_count")
-       .joins("LEFT JOIN #{data} ON #{data}.repo_id = hubstats_repos.id AND #{data}.#{began_time} > '#{time}'")
+       .select("IFNULL(COUNT(DISTINCT hubstats_deploys.id),0) AS deploy_count")
+       .joins("LEFT JOIN hubstats_deploys ON hubstats_deploys.repo_id = hubstats_repos.id AND (hubstats_deploys.deployed_at BETWEEN '#{start_date}' AND '#{end_date}')")
```

Comment:
We should probably sanitize this so it's not a sql injection vulnerability. It's a little gross with joins since rails does do it for you but it would look something like this:
```
.joins(sanitize_sql_array(["LEFT JOIN hubstats_deploys ON hubstats_deploys.repo_id = hubstats_repos.id AND (hubstats_deploys.deployed_at BETWEEN ? AND ?)", start_date, end_date])
```